bedrock/mozorg/views.py

```diff
@@ -234,7 +234,40 @@ class ContributeIndex(l10n_utils.LangFilesMixin, TemplateView):
     template_name = 'mozorg/contribute/index.html'
 
 
+class ContributeSignupNew(TemplateView):
+
+    def get_template_names(self):
+        variant = self.request.GET.get('v', '')
```

Comment:
Could avoid duplicating this logic by adding `variant` as an instance variable of the `ContributeSignupNew` class and passing the value from `contribute_signup`.

```python
class ContributeSignupNew(TemplateView):
    variant = None

    def get_template_names(self):
        template = 'mozorg/contribute/signup-alternate.html'
        
        if self.variant == '2':
        # no changes past this point
```

```python
# in contribute_signup
return ContributeSignupNew.as_view()(request, variant)
```